File: src/main/java/com/steam/reviews/steamreviews/domain/GameInfo.java
package com.steam.reviews.steamreviews.domain;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.io.Serializable;

@Data
@AllArgsConstructor
public class GameInfo  implements Serializable {
    private String name;
    private String imageUrl;
    private int id;
}

File: src/main/steam-reviews-app/src/app/domain/GameInfo.ts
export class GameInfo {
  private _name: string = "";

  get name(): string {
    return this._name;
  }

  private _imageUrl: string = "";

  get imageUrl(): string {
    return this._imageUrl;
  }

  private _id: number = 0;

  get id(): number {
    return this._id;
  }
}
